### DISCUSSION TITLE
Passphrase SSh Key Problem

### DISCUSSION THREAD

**Discussion by Aishaakin** [community/community#47757] [New to GitHub]
I am having a problem entering my passphrase key on my local machine. so i want to ask how can i change my passphrase key password.

**Comment by queenofcorgis** [comment]
@Aishaakin - are you referring to your GitHub password, if so this [doc](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/updating-your-github-access-credentials) will take you through step-by-step 👍🏼 . 

If you're referring to your SSH key - then I'm going to point you to this [documentation](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent). 

Good luck!

**Reply by Aishaakin** [reply]
Thanks for your help, means a lot ot me.

**Comment by airtower-luna** [comment]
The [`-p` option of `ssh-keygen`](https://manpages.debian.org/bullseye/openssh-client/ssh-keygen.1.en.html#p) should be what you need if you just want to change the passphrase. :slightly_smiling_face:

**Reply by Aishaakin** [reply]
Thanks for your help, means a lot ot me.
